superset-frontend/src/explore/components/AdhocMetricEditPopoverTitle.jsx

```diff
@@ -18,7 +18,8 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormControl, OverlayTrigger, Tooltip } from 'react-bootstrap';
+import { FormControl } from 'react-bootstrap';
+import Tooltip from 'src/common/components/Tooltip';
```

Comment:
NIT, but wondering how you feel about this being a default export. I kind of prefer named exports, but I won't lose much sleep over it ;)

Reply:
Hmm, I see that we have mixed export styles in `common/components` - we use both default and named exports. I also prefer named exports, so I refactored Tooltip and it's usages, but I think that we should do the same for other components in the future.